lte/gateway/c/core/oai/tasks/nas/api/mme/mme_api.hpp

```diff
@@ -156,32 +156,40 @@ status_code_e mme_api_unsubscribe(bstring apn);
 }
 #endif
 
-status_code_e mme_api_notify_imsi(const mme_ue_s1ap_id_t id,
-                                  const imsi64_t imsi64);
-
 status_code_e mme_api_notify_new_guti(const mme_ue_s1ap_id_t ueid,
                                       guti_t* const guti);
 
+/*TODO: These declarations are temporarily moved to emm_headers.hpp file to
```

Comment:
Could you mention the issue number, https://github.com/magma/magma/issues/13096
The care should be taken while handling this issue